providers/aws/inputsources/shared/masterdata.ftl

```diff
@@ -1053,6 +1053,9 @@
               "cmk": {
                 "Engine": "cmk"
               },
+              "accountcmk" : {
+                "Engine" : "cmk-account"
```

Comment:
Is an account level cmk new? I've probably forgotten but I don't recall the use case...

Reply:
Yes it is. It's part of this PR and allows manageCrypto to find the account level keys ( if they still exist ) using the buildBlueprint